components/mime.rst

```diff
@@ -0,0 +1,577 @@
+.. index::
+   single: MIME
+   single: MIME Messages
+   single: Components; MIME
+
+The Mime Component
+==================
+
+    The MIME component allows manipulating the MIME messages used to send emails.
+
+Installation
+------------
+
+.. code-block:: terminal
+
+    $ composer require symfony/mime
+
+Alternatively, you can clone the `<https://github.com/symfony/mime>`_ repository.
+
+.. include:: /components/require_autoload.rst.inc
+
+Introduction
+------------
+
+`MIME`_ (Multipurpose Internet Mail Extensions) is an Internet standard that
```

Comment:
Do we need to add a note about the fact that you can also create "simple"/non MIME emails with the components?

Reply:
I don't understand this comment. What's a non-MIME message? Also, in the Wikipedia we can read that:

> Virtually all human-written Internet email and a fairly large proportion of automated email is transmitted via SMTP in MIME format.